clang/lib/AST/ItaniumMangle.cpp

```diff
@@ -2445,6 +2445,46 @@ static bool isTypeSubstitutable(Qualifiers Quals, const Type *Ty,
   return true;
 }
 
+namespace {
+using DeclContextDesc = std::pair<clang::Decl::Kind, StringRef>;
```

Comment:
Perhpas better as an aggregate instead of a 'pair'?  Would probably make the below code easier to read.

(struct DeclContextDesc { clang::Decl::Kind kind; StringRef Scope; };)

Reply:
Done